app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.db_connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, second_name: str) -> None:
+        self.db_connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, second_name)"
+            f" VALUES (?, ?)",
+            (first_name, second_name)
+        )
+
+        self.db_connection.commit()
+
+    def read_all(self) -> list:
```

Comment:
The method `read_all` should be renamed to `all` to match the task requirement: "a method that returns a list of `Actor` instances from DB".